test/itkMontagePCMTestSynthetic.cxx

```diff
@@ -39,7 +39,7 @@ class HyperSphereImageSource : public itk::Object
   itkNewMacro(Self);
 
   /** Run-time type information (and related methods). */
-  itkTypeMacro(Image, Object);
+  itkTypeMacro(HyperSphereImageSource, Object);
```

Comment:
As a side note, the new macro definition (https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Core/Common/include/itkMacro.h#L457) does not take the superclass name. So, it maybe a good idea to update the calls in this module if you have the bandwidth as a separate commit.

Reply:
:+1: resolving then.